src/main/java/frc/team2412/robot/commands/LED/LightsCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.team2412.robot.commands.LED;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+import frc.team2412.robot.subsystems.LEDSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class LightsCommand extends Command {
+	private final LEDSubsystem ledSubsystem;
+	private final IntakeSubsystem intakeSubsystem;
+	private final LauncherSubsystem launcherSubsystem;
+
+	public LightsCommand(
+			LEDSubsystem ledSubsystem,
+			IntakeSubsystem intakeSubsystem,
+			LauncherSubsystem launcherSubsystem) {
+		this.ledSubsystem = ledSubsystem;
+		this.intakeSubsystem = intakeSubsystem;
+		this.launcherSubsystem = launcherSubsystem;
+		addRequirements(ledSubsystem);
+	}
+
+	@Override
+	public void initialize() {
+		ledSubsystem.setRED_LED();
+	}
+
+	@Override
+	public void execute() {
+		if (launcherSubsystem.isAtSpeed()) { // Checks if launcher is ready
+			ledSubsystem.setGREEN_LED();
+		} else if (intakeSubsystem.indexSensorHasNote()) { // Checks if note is in feeder
+			ledSubsystem.setBLUE_LED();
+		} else if (intakeSubsystem.isIntakeOn()) { // Checks if intake is on
+			ledSubsystem.setYELLOW_LED();
+		} else { // Everything else including intake off
+			ledSubsystem.setRED_LED();
+		}
```

Comment:
Is this logic correct?  Should it be green if it is up to speed but does not have a note?
Should there be a color difference between a note just entering index and a note in position to be launched?

Reply:
To be green it has been updated to check if it has a note as well. New color violet has been added to check if the feeder sensor detects a note.